app/src/app/components/contents/packets/PacketDetails.tsx

```diff
@@ -31,7 +32,13 @@ export default function PacketDetails() {
     const getHashOfHtmlFile = () => {
         return hasFiles()
             ? packet.files.find(file => file.path.split(".").pop() === "html")
-            : undefined;
+            : null;
+    };
+
+    const getPath = () => {
```

Comment:
Might be worth making this more explicit for clarity

```suggestion
    const getHtmlFilePath = () => {
```